### ISSUE TITLE
Option to do different damage by player or entity

### ISSUE THREAD

**Issue by temoya1** [WeaponMechanics/MechanicsMain#110]
### Description

Option to do different damage by player or entity

### Configuration

```yaml
  Damage:
    Player_Damage: <amount>
    Entity_Damage: <amount>
```

**Comment by CJCrafter** [comment]
This will be added in the config.yml damage section, as it is too "extensive" to be added to every weapon

**Comment by CJCrafter** [comment]
Fixed in next update 1.8.6-BETA
